src/firebase/userDb.js

```diff
@@ -2,13 +2,14 @@ import { db } from './firebaseConfig';
 import { collection, addDoc, getDocs, setDoc, where, query, doc } from 'firebase/firestore';
 import property_json from './data/propertyData';
 
+const allUsers = "user_mock_data";
```

Comment:
Nitpicky: I suggest to move this into getAllUsers() function to make it a local variable since currently no other functions are using this var.

Reply:
fixed